scss/_utilities_baseline-grid.scss

```diff
@@ -40,5 +40,9 @@
     position: fixed;
     right: $sp-unit * 3;
     z-index: 201;
+
+    .p-switch__label {
```

Comment:
Good catch. 
But because the switch doesn't have any background if we make the text always black, it may not be visible on some screen sizes (if it appears on top of the dark background of the page).

This is not necessarily in scope of this PR, so leave it like this. Maybe we need to add the background to the "toolbar" with the switches, or something.

Reply:
Like here:

<img width="543" alt="image" src="https://github.com/canonical/vanilla-framework/assets/83575/bd0ce921-5779-4501-b771-ec39e410d24b">